app/v1/controllers/LiabilitiesController.scala

```diff
@@ -71,11 +74,11 @@ class LiabilitiesController @Inject()(val authService: EnrolmentsAuthService,
 
      result.leftMap { errorWrapper =>
 
-        val correlationId = getCorrelationId(errorWrapper)
-        val leftResult = errorResult(errorWrapper).withApiHeaders(correlationId)
-        logger.warn(ControllerError(endpointLogContext ,vrn, request, leftResult.header.status, errorWrapper.error.message))
+       val resCorrelationId: String = errorWrapper.correlationId
+       val leftResult = errorResult(errorWrapper).withApiHeaders(resCorrelationId)
+       logger.warn(ControllerError(endpointLogContext ,vrn, request, leftResult.header.status, errorWrapper.error.message, resCorrelationId))
 
-        auditService.auditEvent(AuditEvents.auditLiabilities(correlationId,
+       auditService.auditEvent(AuditEvents.auditLiabilities(correlationId,
```

Comment:
I think this should be

```suggestion
       auditService.auditEvent(AuditEvents.auditLiabilities(resCorrelationId,
```